maplibre_gl_platform_interface/lib/src/method_channel_maplibre_gl.dart

```diff
@@ -1,6 +1,6 @@
 part of '../maplibre_gl_platform_interface.dart';
 
-class MethodChannelMaplibreGl extends MapLibreGlPlatform {
+class MethodChannelMapLibreGl extends MapLibreGlPlatform {
```

Comment:
Lets go with `MapLibreGlMethodChannel` for this?

Reply:
How about shorten it down to `MapLibreMethodChannel`?

Reply:
I don't mind, we have a mix of names with and without the gl suffix anyways.